staging/src/k8s.io/apiserver/pkg/util/openstack/flavors.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2021 The Arktos Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openstack
+
+import "fmt"
+
+var flavors = map[string]FlavorType{}
+var flavorList = []*FlavorType{}
+
+// slim down version of the openstack flavor data structure
+type FlavorType struct {
+	Id          int
+	Name        string
+	MemoryMb    int
+	Vcpus       int
+	RootGb      int
+	EphemeralGb int
+}
+
+// for 130 release, only READ operation with static list of flavors
+func initFlavorsCache() {
```

Comment:
Question: I did not find the usage of initFlavorsCache and initImagesCache. When will they be called and initialize the cache?